src/ducks/account/helpers.js

```diff
@@ -139,40 +144,74 @@ const isWithin6Months = () => {
   return date => isAfter(date, SIX_MONTHS_AGO)
 }
 
-export const buildHealthReimbursementsVirtualAccount = transactions => {
-  const healthExpensesFilter = overEvery(
+const buildReimbursementsVirtualAccount = (filter, options) => transactions => {
```

Comment:
Could you add a docstring ? I would be particularly interested in what goes into options.
I see there is 

- translationKey
- id
- categoryId

Reply:
Here it is https://github.com/cozy/cozy-banks/pull/1773/commits/ce6199e464ffbce057215103a82cd4ad76181874